primitives/src/pcs/univariate_kzg/mod.rs

```diff
@@ -1296,10 +1296,7 @@ mod tests {
     #[cfg(feature = "icicle")]
     mod icicle {
         use super::*;
-        use crate::{
-            icicle_deps::{curves::*, *},
-            pcs::univariate_kzg::icicle::GPUCommittable,
-        };
+        use crate::{icicle_deps::curves::*, pcs::univariate_kzg::icicle::GPUCommittable};
```

Comment:
My local compiler is complaining about missing imports `crate::icicle_deps::warmup_new_stream`

Reply:
fixed in [9b0540c](https://github.com/EspressoSystems/jellyfish/pull/522/commits/9b0540cd25394057e3d9fb5cf9ea0bbd3a02429b)